### ISSUE TITLE
doesn't invoke a custom webpack config function

### ISSUE THREAD

**Issue by nksfrank** [ai/size-limit#315]
trying to set up size-limit and use my `webpack.config.js` that exports a function for building the webpack config object.

size-limit doesn't seem to accept a webpack config as a function.

```javascript
const clientConfig = (argv, env) => ({
  mode: env.mode,
  devtool: env.mode === 'production' ? 'source-map' : 'eval',
  name: 'client',
  /* ... */
});

console.log('file is read');
module.exports = (argv, env) => {
  console.log('function is invoked', argv, env);
  return clientConfig(argv, env);
};
```
the `function is invoked` is never printed

**Comment by ai** [comment]
Yes, it doesn’t.

Send PR if you have an idea how to add support.

**Comment by ai** [comment]
The [fix](https://github.com/ai/size-limit/pull/339) was released in 10.0.2.
